base/markdown/render/html.jl

```diff
@@ -5,7 +5,7 @@ include("rich.jl")
 function withtag(f, io::IO, tag, attrs...)
     print(io, "<$tag")
     for (attr, value) in attrs
-        print(io, " $attr=\"$value\"")
+        print(io, " ", htmlescape(attr), "=\"", htmlescape(value), "\"")
```

Comment:
Do attributes ever need to be escaped? As far as I know something special characters aren't valid attribute names anyway, and since the values are quoted they shouldn't need escaping.